github-runner-provisioner/internal/aws/runner.go

```diff
@@ -0,0 +1,28 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/datawire/infra-actions/github-runner-provisioner/internal/utils"
+)
+
+const LabelTag = "label"
+const NameTag = "app"
+const ownerTag = "owner"
+const repoTag = "repo"
```

Comment:
Why are some of these camelCase and others Pascal?

Reply:
capitalized variabels are public. Only NameTag is used outside of this package so it's the only one exposed.